src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND_OF_NUMBER = 100;
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return new Ball(colorSupplier.getRandomColor(), (random.nextInt(BOUND_OF_NUMBER) + 1));
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(BOUND_OF_NUMBER) + 1);
```
these brackets are redundant. Also better create separate local variables for getting parameters and pass only names to the constructor. It's increase readability